WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -54,44 +44,28 @@
 
               <td class="wca-id">
                 <% if registration.wca_id %>
-                  <%= render "shared/wca_id", wca_id: registration.wca_id %>
+                  <%= link_to registration.wca_id, "/results/p.php?i=#{registration.wca_id}" %>
```

Comment:
It should also handle `nil`, so we can get rid of all these:

            <% if registration.select_wca_id %>
              <%= render "shared/wca_id", wca_id: registration.select_wca_id %>
            <% end %>